include/seqan3/io/sequence_file/input.hpp

```diff
@@ -426,21 +426,21 @@ struct sequence_file_input_default_traits_aa : sequence_file_input_default_trait
  *
  * Note that for this to make sense, your storage data types need to be identical to the corresponding column types
  * of the file. If you require different column types you can specify you own traits, see
- * seqan3::sequence_file_input_traits_concept.
+ * seqan3::SequenceFileInputTraits.
  *
  * ### Formats
  *
  * TODO give overview of formats, once they are all implemented
  */
 
 template <
-    sequence_file_input_traits_concept                       traits_type_        = sequence_file_input_default_traits_dna,
+    SequenceFileInputTraits                                  traits_type_        = sequence_file_input_default_traits_dna,
```

Comment:
```suggestion
    SequenceFileInputTraits                    traits_type_        = sequence_file_input_default_traits_dna,
```
You can change the indentation here